### DISCUSSION TITLE
Do we really want AI be humorous/funny/considerate/natural and even humanlike?

### DISCUSSION THREAD

**Discussion by lets-make-safe-ai** [lets-make-safe-ai/make-safe-ai#16] [General]
Do we really want AI be humorous/funny/considerate/natural and even humanlike?

**Comment by ChenChenDu** [comment]
这很重要！

**Comment by mochou37** [comment]
我们需要一个朋友，一个能耐心的倾听的朋友，并给予一定的建议。我们还可以探讨奇闻乐事，丰富知识，交流精神世界！这是非常乐观的！

**Comment by chirag127** [comment]
yes, I like humorous ai.
